Foundation/NSPropertyList.swift

```diff
@@ -117,6 +138,8 @@ internal func _expensivePropertyListConversion(input : AnyObject) -> Any {
         return true
     } else if input === kCFBooleanFalse {
         return false
+    } else if CFGetTypeID(input) == _CFKeyedArchiverUIDGetTypeID() {
```

Comment:
__NSCFType is lacking _cfTypeID, is this a bug? I have fixed for now just to check "input is __NSCFType" above. If you would prefer me to add _cfTypeID to __NSCFType in NSSwiftRuntime.swift let me know.